mitielib/include/mitie.h

```diff
@@ -446,6 +479,23 @@ extern "C"
               there is some error that prevents us from writing to the given file.
     !*/
 
+    MITIE_EXPORT int mitie_save_named_entity_extractor_pure_model (
+        const char* filename,
+        const mitie_named_entity_extractor* ner
+    );
+    /*!
+        requires
+            - filename == a valid pointer to a NULL terminated C string
+            - ner != NULL
+        ensures
+            - Saves part of the ner object to disk in a file with the given filename.  
+              Specifically, it saves everything except the feature extractor. 
+              Once this function finishes you will be able to read the ner object 
+              from disk by calling 
+              mitie_load_named_entity_extractor(filename,feature_extractor_filename).
```

Comment:
Shouldn't this refer to the pure version?   Also, it's probably a good idea if the pure load functions reference these functions so it's clear what it is the load functions are capable of loading.